src/Worker.php

```diff
@@ -110,6 +111,11 @@ public function handle(Request $request, RequestContext $context): void
         } catch (Throwable $e) {
             $this->handleWorkerError($e, $sandbox, $request, $context, $responded);
         } finally {
+            $sandbox->flush();
+
+            $this->app->make('view.engine.resolver')->forget('blade');
```

Comment:
Is there a way to forget all? If not, could it be added to Laravel today. Otherwise, other custom things will still cause leaks.